dockerfiles/train_model.dockerfile

```diff
@@ -3,9 +3,7 @@ FROM hatespeech-base
 
 COPY pyproject.toml pyproject.toml
 COPY mlops_project/ mlops_project/
-COPY data/ data/
```

Comment:
How can we train the model without having the data in the docker container? @dgloukhman